### ISSUE TITLE
Adding key support next/prev

### ISSUE THREAD

**Issue by GoogleCodeExporter** [AltynbekKZ/galleriffic#20]
```
Hiya,

Just an idea for improvement could be key support like when you hit the
arrow right you go 1 image to the right or something like that. Pretty much
like facebook. That would be aboslutely insane! And I am sure i'd donate
then :D

And in case you don't know them I'll just give you some basic javascript
keycodes.
Arrow up = 38 (Start slideshow prehaps?)
Arrow Down = 40 (Stop slideshow?)
Arrow Right = 39 (Next image)
Arrow Left = 37 (Previous image)

Keep up the awesome work!

Cheers
Kasper


```

Original issue reported on code.google.com by `Kasper.b...@gmail.com` on 28 Aug 2009 at 9:53

**Comment by GoogleCodeExporter** [comment]
```
A more detailed list on other keycodes -
http://www.aspdotnetfaq.com/Faq/What-is-the-list-of-KeyCodes-for-JavaScript-KeyD
own-KeyPress-and-KeyUp-events.aspx

```

Original comment by `Kasper.b...@gmail.com` on 28 Aug 2009 at 9:56

**Comment by GoogleCodeExporter** [comment]
```
This has been added to the roadmap.
```

Original comment by `trentfoley` on 17 Oct 2009 at 9:30
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Keyboard nav is now available in release 2.0. 
```

Original comment by `trentfoley` on 31 Oct 2009 at 4:05
- Changed state: **Released**
